build.gradle

```diff
@@ -79,7 +79,9 @@ configure(subprojects) {
         slf4jVersion = '1.7.30'
         sparkVersion = '2.5.2'
 
-        os = osdetector.os == 'osx' ? 'mac' : osdetector.os == 'windows' ? 'win' : osdetector.os
+        def osName = osdetector.os == 'osx' ? 'mac' : osdetector.os == 'windows' ? 'win' : osdetector.os
+        def osArch = System.getProperty("os.arch").toLowerCase()
+        os = (osName == 'mac' && (osArch.contains('aarch64') || osArch.contains('arm'))) ? 'mac-aarch64' : osName
```

Comment:
How is "mac-aarch64" used or why is it necessary? I don't see it used anywhere.

Reply:
The JavaFX packages use `mac-aarch64` to identify the MacOS aarch64 JARs:

https://github.com/haveno-dex/haveno/blob/62228625cb98341d080080dddf80a0d679f91721/gradle/verification-metadata.xml#L2412
https://github.com/haveno-dex/haveno/blob/62228625cb98341d080080dddf80a0d679f91721/gradle/verification-metadata.xml#L2429
https://github.com/haveno-dex/haveno/blob/62228625cb98341d080080dddf80a0d679f91721/gradle/verification-metadata.xml#L2446
https://github.com/haveno-dex/haveno/blob/62228625cb98341d080080dddf80a0d679f91721/gradle/verification-metadata.xml#L2460

If I remember correctly, I needed to add this in order to properly resolve the `javafx-*-21.0.2-mac-aarch64.jar`
I'll remove it and test if building still works on Apple Silicon based Mac